bin/nvcc_wrapper

```diff
@@ -392,6 +392,9 @@ if [ $first_xcompiler_arg -eq 0 ]; then
   nvcc_command="$nvcc_command -Xcompiler $xcompiler_args"
 fi
 
+#Replace all commas in xcompiler_args with a space for the host only command
+xcompiler_args=${xcompiler_args//,/" "}
```

Comment:
I don't believe this is correct. We only can do this if "host_only" is true, otherwise this will break when cuda files are build.

Reply:
`xcompiler_args` is not used anymore after setting up `host_command` and `host_command` is only used in `host-only` mode.

Reply:
ah right. You are changing it after it is used for the nvcc_command. OK.